### PR TITLE
add instructions for using Docker for local jekyll build

### PR DESCRIPTION
**Pull request by orbeckst**
Update to the README: From hackathon: How to build the docs locally using a docker container because somehow all the ruby packages live in the 5th circle of dependency hell.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,3 +140,48 @@ with
 
 
     bundle update github-pages
+
+
+#### Using docker container for local builds ####
+
+**Problems with installing jekyll/github-pages?** Try a docker
```
Comment (IAlibay) [inline README.md:147]: I would frame this as an alternative approach - actually maybe it should be the primary one?

**Review by IAlibay — COMMENTED** [review]
I'm not going to block because lgtm otherwise, but maybe it should be best practices to just do this.

File: README.md (hunk 1 above)
Comment (orbeckst) [reply README.md:147]: ok, I'll rewrite

Comment (orbeckst) [reply README.md]: See 5d95a0f

**Review by IAlibay — APPROVED** [review]
lgtm!

**Comment by hmacdope** [comment]
Thank god for this, I haven't been able to get a working build after several attempts.

**Pull request merged**